practices/development.md

```diff
@@ -2,105 +2,90 @@
-* [Tasks](coordination:tasks) are concrete actions to take, and should be accomplishable in a single work session by one person.
-* The [Activity Backlog](coordination:active-backlog) contains lists of **tasks** that we are currently working on. This defines 2i2c's current activity.
-
-(coordination:project-backlog)=
-## Project Backlogs
-
-The Project Backlog contains all of the deliverables across our projects that we wish to work on quickly.
-These are organized into a few columns, representing the _state_ of each deliverable.
-Here is a common column structure:
-
-- {guilabel}`Needs Discussion/Refinement`: Deliverables that are high-priority but un-refined. Our goal should be having discussion and doing research in order to get these deliverables ready for work.
-- {guilabel}`Ready to Work`: Deliverables that are well-scoped and have a clear path forward, and are thus ready to implement. As deliverables in {guilabel}`In progress` are completed, we should replace them with deliverables from this column. Generally speaking, deliverables near the top have higher priority than those at the bottom.
-- {guilabel}`In progress`: Deliverables that we are currently working towards. This means that we should be generating [Tasks](coordination:tasks) in our [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to complete this deliverable (more on this below).
-- {guilabel}`Blocked/Waiting`: Deliverables for which we are waiting for some action that is out of our team's immediate control.
-- {guilabel}`Done`: Deliverables that have been completed. We should close these issues and celebrate the improvements that we have made!
-
-:::{note}
-In general, there should be only two or three deliverables per team member, per column on this board.
-It should not become so heavily-populated that it is hard to keep track of deliverables!
+:::{admonition} Helpful links
+👉 [Here is a link to all 2i2c GitHub Issues that have been assigned to you](https://github.com/issues?q=is%3Aissue+is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+assignee%3A%40me+user%3A2i2c-org+)
+👉 [Here's a link to see all Pull Requests for which your review is requested](https://github.com/issues?q=is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+user%3A2i2c-org+type%3Apr+review-requested%3A%40me+)
 :::
 
-There are two Project Backlogs at 2i2c:
-
-- The [2i2c Development Projects Backlog](https://github.com/orgs/2i2c-org/projects/7?fullscreen=true) contains deliverables for our development-focused projects.
-- The [Organizational Project Backlog](https://github.com/orgs/2i2c-org/projects/8?fullscreen=true) contains deliverables that are organization-wide and not related to development.
+## Cadence of work
 
+Our team works in one week cycles.
+Each cycle begins on Monday with our [team deliverables review meeting](meetings:deliverables-review).
```

Comment:
If we are having the deliverable meeting on Tue, then the cycle should start on Tue, right?